From: Etienne Millon <me@emillon.org>
Date: Sat, 18 Aug 2012 17:41:58 +0200
Subject: Fix format string function with a non-literal

As the string is not a literal (and potentially not trusted), use a "%s" format
string.
---
 src/RageInput.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/RageInput.cpp
+++ b/src/RageInput.cpp
@@ -66,7 +66,7 @@
 
 	// If no input devices are loaded, the user won't be able to input anything.
 	if( apDevices.size() == 0 )
-		LOG->Warn( NO_INPUT_DEVICES_LOADED.GetValue() );
+		LOG->Warn( "%s", NO_INPUT_DEVICES_LOADED.GetValue().c_str() );
 }
 
 void RageInput::Update()
